doc/tutorials/quantization.md

```diff
@@ -0,0 +1,60 @@
+# Post-Training Quantization
+
+The quantization is used to reduce the  the precision of the weights and activations to other (lower) bit-widths. This is usually useful when working on limited hardware, where a reduced model size is required.
+
+You can choose your conversion factor with the `LinearQuantization` and `ScaleQuantization `functions. They both map the weights from floating-point weights to the int8 range, but they use different ways to calculate and apply the conversion.
+
+**Linear Quantization:**
+The scale calculation goes as follows:
+$$
+\text{scale} = \frac{(2^{\text{numBits} - 1} - 1)}{\text{maxAbs}}
+$$
```

Comment:
I don't think that the Markdown parser will successfully parse the LaTeX here, so it might be better to write it as a simple formula enclosed in backticks.  (I know, it is uglier like that, but we are limited by Markdown in what it can do.)